Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:alpine
+
+WORKDIR /home/node/app
+COPY package*.json ./
+COPY . .
+# EVN NODE_ENV=development
+# ENV NODE_PATH=./build
```

Comment:
why do we need this commented code here please?

Reply:
safe to remove